python/triton/experimental/gluon/language/amd/cdna3/__init__.py

```diff
@@ -148,10 +170,10 @@ def buffer_atomic_rmw(op, ptr, offsets, value, mask=None, sem=None, scope=None,
         op (str) : The operator to be executed atomically.
         ptr (pointer to scalar): Global memory scalar base pointer to load from.
         offsets (tensor): Offsets tensor for the load operation.
-        value (tensor): Another opearand of `op`.
+        value (tensor): Another operand of `op`.
         mask (tensor, optional): Mask tensor for predicated loads. Defaults to None.
-        sem (str, optional): Memory Semantic Descriptor. Default is None.
-        scope (str, optional): Memory Sync Scope. Default is None.
+        sem (str, optional): Memory Semantic Descriptor. Default is None and it will be mapped to `accquire-and-release` mode or happens-before in other words.
```

Comment:
What do you want to additionally explain with "or happens-before in other words"? Just say "Default is None which means acq_rel memory semantic" is enough.